### ISSUE TITLE
1.10.2 Crash With Latest InfinityLib

### ISSUE THREAD

**Issue by Redlotus99** [InfinityRaider/InfinityLib#17]
I seem to be crashing off my server when placing or removing some Agricraft ducts.

Here is the crash report.

http://pastebin.com/WCiBJGa8

**Comment by Redlotus99** [comment]
I've just updated to the newest InfinityLib and Agricraft and still seem to have some issues. It seems that now with just crop sticks the client is crashing after a bit of time. I'll see if I can get more information.

**Comment by theCorvid** [comment]
I'm getting the same crash. [Paste here](http://pastebin.com/0LQvBpXx).

Started having the problem for the first time when placing Agricraft irrigation channels, at which point there were only some channels and some water tanks in the area. I removed the tanks and the channels, and then had this crash later randomly - only crop sticks with wheat plants in them were in the area at this time.

**Comment by theCorvid** [comment]
Update on the issue:

Something really strange happened when I was updating to the latest version of InfinityLib and Agricraft. I shut the server down and updated client and server, and logged back in, and oddly, the irrigation pipes that I had previously removed were back in the same place again. I asked my friend - who had been playing with me - about it, and he said he didn't know what I was talking about, and that they had been there the whole time. So, basically, I had removed the pipes and they had visually disappeared on my client but they hadn't disappeared server-side. Maybe this is relevant to why rendering crashes are happening?

**Comment by Redlotus99** [comment]
Seems similar. I actually thought I had my irrigation pipes working along with the sprinklers until I noticed the sprinklers were not visible, only water was visible and spraying the area.  After a bit of time I noticed that my Minecraft started running horribly. After I looked into that more, I noticed Minecraft eating away at memory and it spiked to the point I had to ctrl-alt-del out.

After loading the game back up I decided to remove the irrigation, but oddly when I deleted it and then placed a block where sprinklers were at previously the sprinkler would oddly show up as I had not deleted it. So after removing all of that (including Crop Sticks) and a server reboot my Minecraft seemed to run fine.
